### PR TITLE
Calendar: support ID prop

### PR DESCRIPTION
**Pull request by micahgodbolt**
v8 calendar does not support ID prop (and v7 does). This fix helps with migration as well as consistency between components.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/48df172dd62d2bc71409c8cdb3967ea500dcd999'>48df172dd62d2bc71409c8cdb3967ea500dcd999</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/BaseButton-mount_base.html">1373</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/BaseButton-mount.html">1343</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Breadcrumb-mount_base.html">3300</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Breadcrumb-mount.html">3372</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Checkbox-mount_base.html">2981</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Checkbox-mount.html">3039</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/CheckboxBase-mount_base.html">2733</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/CheckboxBase-mount.html">2699</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/ChoiceGroup-mount_base.html">5098</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/ChoiceGroup-mount.html">5057</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/ComboBox-mount_base.html">1425</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/ComboBox-mount.html">1447</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/CommandBar-mount_base.html">10867</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/CommandBar-mount.html">10912</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/ContextualMenu-mount_base.html">12352</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/ContextualMenu-mount.html">12555</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/DefaultButton-mount_base.html">1588</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/DefaultButton-mount.html">1577</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/DetailsRow-mount_base.html">4207</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/DetailsRow-mount.html">4108</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/DetailsRowFast-mount_base.html">4126</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/DetailsRowFast-mount.html">4247</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/DetailsRowNoStyles-mount_base.html">4121</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/DetailsRowNoStyles-mount.html">3974</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Dialog-mount_base.html">3569</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Dialog-mount.html">3601</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/DocumentCardTitle-mount_base.html">667</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/DocumentCardTitle-mount.html">671</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Dropdown-mount_base.html">3693</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Dropdown-mount.html">3667</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/FocusTrapZone-mount_base.html">2338</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/FocusTrapZone-mount.html">2381</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/FocusZone-mount_base.html">2293</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/FocusZone-mount.html">2229</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/GroupedList-mount_base.html">59874</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/GroupedList-mount.html">67100</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/GroupedList-virtual-rerender_base.html">28154</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/GroupedList-virtual-rerender.html">28788</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">104629</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/GroupedList-virtual-rerender-with-unmount.html">105263</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/GroupedListV2-mount_base.html">643</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/GroupedListV2-mount.html">642</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/GroupedListV2-virtual-rerender_base.html">610</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/GroupedListV2-virtual-rerender.html">605</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">666</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">626</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/IconButton-mount_base.html">2186</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/IconButton-mount.html">2198</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Label-mount_base.html">823</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Label-mount.html">836</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Layer-mount_base.html">4878</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Layer-mount.html">4921</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Link-mount_base.html">963</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Link-mount.html">946</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/MenuButton-mount_base.html">1922</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/MenuButton-mount.html">1923</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/MessageBar-mount_base.html">2640</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/MessageBar-mount.html">2593</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Nav-mount_base.html">3718</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Nav-mount.html">3714</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/OverflowSet-mount_base.html">1554</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/OverflowSet-mount.html">1558</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Panel-mount_base.html">2918</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Panel-mount.html">2970</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Persona-mount_base.html">1472</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Persona-mount.html">1482</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Pivot-mount_base.html">1868</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Pivot-mount.html">1941</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/PrimaryButton-mount_base.html">1726</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/PrimaryButton-mount.html">1752</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Rating-mount_base.html">8003</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Rating-mount.html">8009</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/SearchBox-mount_base.html">1708</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/SearchBox-mount.html">1726</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Shimmer-mount_base.html">3219</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Shimmer-mount.html">3226</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Slider-mount_base.html">2409</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Slider-mount.html">2439</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/SpinButton-mount_base.html">5362</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/SpinButton-mount.html">5381</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Spinner-mount_base.html">943</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Spinner-mount.html">916</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/SplitButton-mount_base.html">3539</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/SplitButton-mount.html">3588</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Stack-mount_base.html">983</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Stack-mount.html">977</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/StackWithIntrinsicChildren-mount_base.html">2711</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/StackWithIntrinsicChildren-mount.html">2646</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/StackWithTextChildren-mount_base.html">5450</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/StackWithTextChildren-mount.html">5490</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/SwatchColorPicker-mount_base.html">12064</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/SwatchColorPicker-mount.html">11879</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/TagPicker-mount_base.html">3032</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/TagPicker-mount.html">2995</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/TeachingBubble-mount_base.html">94059</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/TeachingBubble-mount.html">93572</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Text-mount_base.html">879</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Text-mount.html">892</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/TextField-mount_base.html">1821</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/TextField-mount.html">1844</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/ThemeProvider-mount_base.html">1729</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/ThemeProvider-mount.html">1727</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/ThemeProvider-virtual-rerender_base.html">1233</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/ThemeProvider-virtual-rerender.html">1240</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2462</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2479</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Toggle-mount_base.html">1267</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/Toggle-mount.html">1286</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/buttonNative-mount_base.html">620</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25013/perf-test/buttonNative-mount.html">638</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: change/@fluentui-react-170d6594-4aee-4504-8885-b70ae2901269.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "chore: support ID prop ",
```
Comment (khmakoto) [inline change/@fluentui-react-170d6594-4aee-4504-8885-b70ae2901269.json:3]:
```suggestion
  "comment": "chore: support ID prop in Calendar",
```

**Review by khmakoto — APPROVED** [review]

File: packages/react/src/components/Calendar/Calendar.base.tsx [hunk 2]
```diff
@@ -326,6 +327,7 @@ export const CalendarBase: React.FunctionComponent<ICalendarProps> = React.forwa
 
     return (
       <div
+        id={id}
```
Comment (GeoffCoxMSFT) [inline packages/react/src/components/Calendar/Calendar.base.tsx:330]: I think v7 CalendarBase never actually applied id to the div.  The forwarded ref has the HTMLDivElement props, but not the ICalendarProps.  I think we could add id to v8, but then ODSP would see an output change when moving to v8.

Comment (micahgodbolt) [reply packages/react/src/components/Calendar/Calendar.base.tsx:330]: I checked it before, and the ID does get passed through. Here's a demo

https://codepen.io/micahgodbolt/pen/PoeRpVo?editors=0111

**Review by GeoffCoxMSFT — APPROVED** [review]

**Review by conniec218 — APPROVED** [review]

**Pull request merged**